GVFS/GVFS.UnitTests.Windows/GVFS.UnitTests.Windows.csproj

```diff
@@ -113,21 +113,14 @@
     <Reference Include="System.Xml" />
   </ItemGroup>
   <ItemGroup>
-    <Compile Include="Windows\Mock\MockInstallerPreRunChecker.cs" />

```

Comment:
These are now moved to GVFS.UnitTests project?

Reply:
yep - as these are no longer Windows specific.